jupyterlab/labapp.py

```diff
@@ -580,6 +580,14 @@ class LabApp(NotebookConfigShimMixin, LabServerApp):
         document will be kept in memory forever.""",
     )
 
+    collaborative_document_save_delay = Int(
```

Comment:
Should it be a `Float`?

Reply:
Done in https://github.com/jupyterlab/jupyterlab/pull/13392